src/main/electron/listener.cljs

```diff
@@ -233,7 +234,17 @@
 
   (js/window.apis.on "syncAPIServerState"
                      (fn [^js data]
-                       (state/set-state! :electron/server (bean/->clj data)))))
+                       (state/set-state! :electron/server (bean/->clj data))))
+
+  (js/window.apis.on "invokeEditorHandler"
+                     (fn [action]
+                       (println "invokeEditorHandler with action:" action)
+                       (case action
+                         "undo" (history/undo! nil)
+                         "redo" (history/redo! nil)
+                         "copy" (editor-handler/shortcut-copy nil)
+                         "cut" (editor-handler/shortcut-cut nil) ;; FIXME this handler relies on arg Event
```

Comment:
Maybe we can change from directly calling the handler to dispatching the event.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent